apps/web/src/components/Table/ResponsiveTable/RowSelection.tsx

```diff
@@ -253,76 +246,73 @@ const Actions = ({
                   })}
                 </Button>
               </span>
-
-              {download?.selection && (
+              {(download?.csv?.enable || download?.doc?.enable) && (
                 <span
                   data-h2-align-items="base(center)"
                   data-h2-display="base(flex)"
                   data-h2-gap="base(0 x.25) l-tablet(0 x.5)"
                 >
-                  <span data-h2-display="base(none) l-tablet(block)">
-                    <Bullet data-h2-display="base(none) l-tablet(block)" />
-                  </span>
-                  <DownloadCsv
-                    data-h2-font-weight="base(400)"
-                    disabled={download.disableBtn}
-                    {...(download.fetching && {
-                      icon: SpinnerIcon,
-                    })}
-                    {...download.selection.csv}
-                    {...actionButtonStyles}
-                  >
-                    {download.selection.label ||
-                      intl.formatMessage({
-                        defaultMessage: "Download CSV",
-                        id: "mxOuYK",
-                        description:
-                          "Text label for button to download a csv file of items in a table.",
-                      })}
-                  </DownloadCsv>
-                </span>
-              )}
-
-              {asyncDownload ? (
-                <span
-                  data-h2-align-items="base(center)"
-                  data-h2-display="base(flex)"
-                  data-h2-gap="base(0 x.5)"
-                >
-                  <span data-h2-display="base(none) l-tablet(block)">
-                    <Bullet data-h2-display="base(none) l-tablet(block)" />
-                  </span>
-                  {asyncDownload}
-                </span>
-              ) : null}
+                  {download?.csv?.enable && (
+                    <>
+                      <span data-h2-display="base(none) l-tablet(block)">
+                        <Bullet data-h2-display="base(none) l-tablet(block)" />
+                      </span>
+                      {download.csv.component || (
+                        <Button
```

Comment:
It would be nice if there was some feedback when clicking the button when no rows are selected. Maybe something like "You must select at least one row"?

## Screen recording
https://github.com/user-attachments/assets/311d8644-ef7c-4bab-9a4e-a2a10be8a87f